src/app/containers/RadioPageBlocks/Blocks/OnDemandHeading/__snapshots__/index.test.jsx.snap

```diff
@@ -89,8 +89,6 @@ exports[`AudioPlayer blocks OnDemandHeading should render correctly 1`] = `
 
 <h1
   class="c0"
-  id="content"
-  tabindex="-1"
 >
   <span
```

Comment:
Is it possible to remove role text and just have all the text without any spans please? The nested spans aren't necessary here as.

Reply:
e.g. <h1 class="c0">Dunia Pagi Ini, 27 April 2020</h1>

Reply:
sorry just seen this is the radio page test...

Reply:
is there a snap test for OD tv?

Reply:
> <h1 class="c0">Dunia Pagi Ini, 27 April 2020</h1>

We need the spans - the title and the date have different styling - the classes on the spans are how we create that

![Screen Shot 2020-06-01 at 13 54 43](https://user-images.githubusercontent.com/8601460/83411202-abae1a00-a40f-11ea-97c6-61e196e8c18c.png)

I have removed the middle span when `aria-hidden` is true though - that is just a visually hidden comma to manipulate how screen readers read it out

Reply:
> is there a snap test for OD tv?

No - there was a pan-simorgh strategy to move away from relying so much on snapshot testing - we do more specific assertion-based testing where possible

Reply:
> sorry just seen this is the radio page test

This is a test for a component that is used by all On Demand pages and episodes

We do have radio tests and radio snapshots, but none of them are changed on this PR, because the radio pages are identical before and after this PR

Reply:
The unneeded role text on the h1 for TV OD pages will reduce our lighthouse a11y score. I'm unsure why we are using the same component for this as OD radio, no doubt there is a good reason. Thanks Ryan.